### PR TITLE
Update Genomics & Galaxy tutorial

### PR DESCRIPTION
**Pull request by pvanheus**
These changes address the issues raised in [#2076]. Outstanding issues are:

1. Need to replace images/genes_human_chr22_rename.png to present a current Galaxy view
2. 'display at UCSC main' seems to no longer exist (at least on usegalaxy.org and usegalaxy.org.au), making part of this tutorial obsolete.

Note that due to the above, this is a Work In Progress.

### CODE REVIEW

File: topics/introduction/tutorials/galaxy-intro-strands/tutorial.md [hunk 1]
```diff
@@ -112,8 +112,7 @@ The Galaxy interface consists of three main parts. The available tools are liste
 > * If this is the first time you have used this Galaxy server, then your history panel will look like the image above.
 >
 > * If your history has datasets in it (there will be one or more green boxes, *datasets*, there) then create a new history:
->     1. Click the **gear icon** at the top of the history panel
->     2. Select the option **Create New** from the menu
+>     * Click the **plus icon** to create a new history
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-strands/tutorial.md:115]: Can you use the `snippets/create_new_history.md` snippet?

File: topics/introduction/tutorials/galaxy-intro-strands/tutorial.md [hunk 2]
```diff
@@ -305,10 +305,10 @@ How might we do this?  Column 6 contains the strand information.  Can we split g
 > ### {% icon hands_on %} Searching for and launching a tool
 >
 > 1. Try the **tools search box**.  Think of terms that might describe what we want to do and type them in the search box.  Do you see anything promising?  Explore a little.
-> 2. If you haven't already searched with it, *enter* `split` in the **tool search box**.  Near the top of the results is
->    - **Filter** data on any column using simple expressions.
+> 2. If you haven't already searched with it, *enter* `filter` in the **tool search box**.  In the results is
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-strands/tutorial.md:308]:
```suggestion
> 2. If you haven't already searched with it, *enter* `filter` in the **tool search box**.  Among the results you should see
```

File: topics/introduction/tutorials/galaxy-intro-strands/tutorial.md [hunk 3]
```diff
@@ -389,7 +389,7 @@ Galaxy excels at answering questions about genomic intervals and different sets
 
 > ### {% icon hands_on %} Hands-on: Genomic Interval Tools
 >
-> * In the tool panel, *open* the **Operate on Genomic Intervals** toolbox.  It's typically past the **NGS** toolboxes.
+> * In the tool panel, *open* the **Operate on Genomic Intervals** toolbox.  It's typically in the **Common Genomics Tools or Text Manipulation** sections.
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-strands/tutorial.md:392]: .org .eu and .org.au all have it under "Common Genomics Tools". Alternatively, I'd have users search for "interval".

Comment (pvanheus) [reply topics/introduction/tutorials/galaxy-intro-strands/tutorial.md:392]: While the "Operate on Genomic Intervals" is in "Common Genomics Tools" on usegalaxy.eu it does not contain the Intersect tool:
image

The procedure in this section encourages users to search through tool panel sections for useful looking tools. I am not sure if this is a good way to explore Galaxy given how many tools there are to search through.

File: topics/introduction/tutorials/galaxy-intro-strands/tutorial.md (hunk 1 above)
Comment (pvanheus) [reply topics/introduction/tutorials/galaxy-intro-strands/tutorial.md]: I've inserted this together with short text above to tell people what they should be doing - the snippet can then explain how to do so.

File: topics/introduction/tutorials/galaxy-intro-strands/tutorial.md (hunk 3 above)
Comment (nsoranzo) [reply topics/introduction/tutorials/galaxy-intro-strands/tutorial.md:392]: > While the "Operate on Genomic Intervals" is in "Common Genomics Tools" on usegalaxy.eu it does not contain the Intersect tool:

Ah, well spotted, it seems a bit weird that the GOPS tools are scattered around 3 sections ("Subtract" is under "Join, Subtract and Group") on UseGalaxy.eu .

> The procedure in this section encourages users to search through tool panel sections for useful looking tools. I am not sure if this is a good way to explore Galaxy given how many tools there are to search through. I'm trying to not make too many changes to the tutorial though.

Indeed, maybe having users search for "interval" (as I was suggesting as alternative) could be an instance-agnostic solution and not a very big change?

Comment (pvanheus) [reply topics/introduction/tutorials/galaxy-intro-strands/tutorial.md]: Yes, good suggestion. I have updated the text accordingly.

File: topics/introduction/tutorials/galaxy-intro-strands/tutorial.md [hunk 4]
```diff
@@ -389,7 +389,7 @@ Galaxy excels at answering questions about genomic intervals and different sets
 
 > ### {% icon hands_on %} Hands-on: Genomic Interval Tools
 >
-> * In the tool panel, *open* the **Operate on Genomic Intervals** toolbox.  It's typically past the **NGS** toolboxes.
+> * The next step is finding overlapping intervals, so search in the tool search box for `interval`. There are many results from this search so you might want to click the *Show sections* button (available on Galaxy servers) to see which sections the tools are organised into.
```
Comment (nsoranzo) [inline topics/introduction/tutorials/galaxy-intro-strands/tutorial.md:392]:
```suggestion
> * The next step is finding overlapping intervals, so type `interval` in the tool search box. There are many results from this search, so you might want to click the *Show sections* button (available on Galaxy servers) to see which sections the tools are organised into.
```

File: topics/introduction/tutorials/galaxy-intro-strands/tutorial.md (hunk 3 above)
Comment (nsoranzo) [reply topics/introduction/tutorials/galaxy-intro-strands/tutorial.md]: Thanks!

**Comment by pvanheus** [comment]
> These changes address the issues raised in [#2076]. Outstanding issues are:
> 
>     1. Need to replace images/genes_human_chr22_rename.png to present a current Galaxy view

Having reviewed the images in the tutorial, I am tending towards not replacing the Galaxy screenshots unless there have been relevant changes to the UI.
> 
>     2. 'display at UCSC main' seems to no longer exist (at least on usegalaxy.org and usegalaxy.org.au), making part of this tutorial obsolete.
> 
@bgruening confirmed that this should have been working on usegalaxy.eu but a config change was necessary. The tutorial can thus stay as it is, but I am a little concerned that it might not work on some servers.

> 
> Note that due to the above, this is a Work In Progress.

**Review by shiltemann — APPROVED** [review]
cool, thanks @pvanheus!

**Pull request merged**